### ISSUE TITLE
Broken on Python 3.6

### ISSUE THREAD

**Issue by e00E** [Drekin/win-unicode-console#34]
```
Python 3.6.0 (v3.6.0:41df79263a11, Dec 23 2016, 08:06:12) [MSC v.1900 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import win_unicode_console
>>> win_unicode_console.enable()
>>> import sys
```
`Python has stopped working` Windows error message and interpreter closes.

**Comment by Drekin** [comment]
One thing is that `win_unicode_console` usually not needed at all on Python 3.6.

**Comment by e00E** [comment]
Youre right, 3.6 changed the console encoding to utf-8. However I also used win-unicode-console to make the default text file encoding utf-8 which is still cp1252 on 3.6 on Windows. I dont know if you want to support this feature.

**Comment by Drekin** [comment]
There may definitely be some other features `win-unicode-console` may be used for, and crashing shouldn't occur in any case, so this should be fixed.

But how did you use WUC to affect default text file encoding? AFAIK WUC doesn't do this.

**Comment by e00E** [comment]
Ah, I was wrong.
I thought a script of mine worked with 3.5 and win-unicode-console but now failed in 3.6 when writing some unicode to a text file.
I just tested everything again and this wasnt case. You are correct that win-unicode-console doesnt do that.
